Src/Newtonsoft.Json.Tests/Newtonsoft.Json.Tests.Roslyn.csproj

```diff
@@ -93,6 +98,9 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0-beta5-build1225" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0-*" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
https://github.com/Microsoft/vstest/issues/472 is tracking removal of this guid.